Dockerfile.ubuntu

```diff
@@ -60,6 +49,3 @@ RUN pip install -r /opt/file-metadata/test-requirements.txt
 
 # Install package
 RUN pip install /opt/file-metadata --upgrade
-
-# Test file-metadata import
-RUN python -c'import file_metadata; print(file_metadata.__version__)'
```

Comment:
It seems there is still some confusion as to what travis does. Travis runs the unittests inside the docker image. So, if the libmagickwand-dev is not there, some of the travis unittests will fail.
Printing the `__version__` will not fail even if the libmagickwand-dev is not installed (Although pip install will fail)